dask_ml/model_selection/_incremental.py

```diff
@@ -119,7 +124,12 @@ def _fit(
     fit_params=None,
     scorer=None,
     random_state=None,
+    verbose: int = 1,
+    prefix="",
 ):
+    logger.info(
+        "[CV%s] train, test examples = %d, %d", prefix, len(y_train), len(y_test)
```

Comment:
What's the type on `y_train`? If it's potentially a dataframe then `len` will be expensive. I'd prefer to omit this, in case we're able to support dataframes eventually.

Reply:
Thanks; this is a meaningless commit message anyways.  `*_train` and `*_test` will be Dask Arrays as per #637.

Now, I'm providing a more meaningful log message on the number of examples in each chunk (which gets at the chunk size balancing, an issue I ran into).